modules/masterbar/masterbar/class-masterbar.php

```diff
@@ -296,13 +296,16 @@ public function add_styles_and_scripts() {
 			false
 		);
 
-		wp_enqueue_script(
-			'a8c_wpcom_masterbar_overrides',
-			$this->wpcom_static_url( '/wp-content/mu-plugins/admin-bar/masterbar-overrides/masterbar.js' ),
-			array( 'jquery' ),
-			JETPACK__VERSION,
-			false
-		);
+		if ( ! apply_filters( 'jetpack_load_admin_menu_class', false ) ) {
```

Comment:
Needs the standard comment pointing out where the filter is documented.
```suggestion
		/** This filter is already documented in modules/masterbar.php */
		if ( ! apply_filters( 'jetpack_load_admin_menu_class', false ) ) {
```

Reply:
Thanks for chiming in @anomiex! Thats is a very good point! Added 6353058